src/utils/events/event-manager.js

```diff
@@ -76,38 +76,46 @@ export default class EventManager {
    */
   on(event, handler) {
     if (typeof event === 'string') {
-      // Special handling for gestural events.
-      const recognizerEvent = EVENT_RECOGNIZER_MAP[event];
-      if (recognizerEvent) {
-        // Enable recognizer for this event.
-        this.manager.get(recognizerEvent).set({enable: true});
-
-        // Alias to a recognized gesture as necessary.
-        const eventAlias = GESTURE_EVENT_ALIASES[event];
-        if (eventAlias && !this.aliasedEventHandlers[event]) {
+      this._onOne(event, handler);
+    } else {
+      // If `event` is a map, call `on()` for each entry.
+      for (const eventName in event) {
+        this._onOne(eventName, event[eventName]);
+      }
+    }
+  }
+
+  _onOne(event, handler) {
+    // Special handling for gestural events.
+    const recognizerEvent = EVENT_RECOGNIZER_MAP[event];
+    if (recognizerEvent) {
+      // Enable recognizer for this event.
+      this.manager.get(recognizerEvent).set({enable: true});
```

Comment:
For readability, It might be good to split this into two lines with a temporary variable to make it more clear what kind of object `get` is returning (I would have to look it up in the code). `get` is a very generically named method, assuming it is set by Hammer and we can't change it?